src/pages/iou/request/step/IOURequestStepDate.js

```diff
@@ -23,21 +27,30 @@ const propTypes = {
     /** Onyx Props */
     /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
     transaction: transactionPropTypes,
+
+    /** The draft transaction that holds data to be persisted on the current transaction */
+    splitDraftTransaction: transactionPropTypes,
 };
 
 const defaultProps = {
     transaction: {},
+    splitDraftTransaction: {},
 };
 
 function IOURequestStepDate({
     route: {
-        params: {iouType, backTo, transactionID},
+        params: {action, iouType, reportID, backTo},
     },
     transaction,
+    splitDraftTransaction,
 }) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
 
+    // In the split flow, when editing we use SPLIT_TRANSACTION_DRAFT to save draft value
+    const isEditingSplitBill = iouType === CONST.IOU.TYPE.SPLIT && action === CONST.IOU.ACTION.EDIT;
```

Comment:
Should we define new variable `const isEditing = action === CONST.IOU.ACTION.EDIT` and use it instead of `action === CONST.IOU.ACTION.EDIT`